go.mod

```diff
@@ -34,6 +35,7 @@ require (
 	github.com/pterm/pterm v0.12.80
 	github.com/rs/cors v1.11.1
 	github.com/sendgrid/sendgrid-go v3.16.0+incompatible
+	github.com/sethvargo/go-limiter v1.0.0
```

Comment:
did you vet this one very much? Does not look like it is widely used.

Reply:
oh, @njhale - lol - this _is_ the one i used in acorn-dns

https://github.com/acorn-io/acorn-dns/pull/40/files

@g-linville it's used in hashicorp projects/products and the author has a good reputation (at least in my experience)

Reply:
Sounds good to me